project_task_work_bottom_edit/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Task Work Bottom Edit
+=====================
+
+This module extends the functionality of Project to support when adding a new line it should be on bottom of the list
```

Comment:
This module extends the functionality of Project to make the lines editable at bottom.